oe_theme.libraries.yml

```diff
@@ -8,6 +8,8 @@ component_library_ec:
     js/ecl_auto_init.js: {}
   dependencies:
     - core/drupal
+    - core/jquery
+    - core/jquery.once
```

Comment:
Tiny remark. Looks like it is not so necessary to add `core/jquery` because `core/jquery.once` already have it self this dependency. But I saw in core similar like here approach.
So it is fine for me. But if you you wish - you could remove `core/jquery`.